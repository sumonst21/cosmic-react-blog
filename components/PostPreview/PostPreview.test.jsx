import post from '../../__mocks__/post';
import { PurePostPreview } from '.';

describe('PostPreview', () => {
  it('should render', () => {
    const component = mount((
      <PurePostPreview
        classes={{
          button: {},
          hero: {},
        }}
        {...post}
      />
    ));
    return expect(toJson(component)).toMatchSnapshot();
  });
});
