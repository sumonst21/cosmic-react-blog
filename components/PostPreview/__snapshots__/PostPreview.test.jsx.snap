// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostPreview should render 1`] = `
<PostPreview
  _id="5afdbf6c44d74b4e924f1d0b"
  bucket="5afdbcd115bef94f7421eb0d"
  classes={
    Object {
      "button": Object {},
      "hero": Object {},
    }
  }
  content="<p>When I orbited the Earth in a spaceship, I saw for the first time how beautiful our planet is. Mankind, let us preserve and increase this beauty, and not destroy it!</p><p>Space, the final frontier. These are the voyages of the Starship Enterprise. Its five-year mission: to explore strange new worlds, to seek out new life and new civilizations, to boldly go where no man has gone before.</p><p>If you could see the earth illuminated when you were in a place as dark as night, it would look to you more splendid than the moon.</p><p>To be the first to enter the cosmos, to engage, single-handed, in an unprecedented duel with nature&mdash;could one dream of anything more?</p><p>We choose to go to the moon in this decade and do the other things, not because they are easy, but because they are hard, because that goal will serve to organize and measure the best of our energies and skills, because that challenge is one that we are willing to accept, one we are unwilling to postpone, and one which we intend to win.</p><p>NASA is not about the &lsquo;Adventure of Human Space Exploration&rsquo;&hellip;We won&rsquo;t be doing it just to get out there in space &ndash; we&rsquo;ll be doing it because the things we learn out there will be making life better for a lot of people who won&rsquo;t be able to go.</p><p>Problems look mighty small from 150 miles up.</p><p>That&#39;s one small step for [a] man, one giant leap for mankind.</p><p>Where ignorance lurks, so too do the frontiers of discovery and imagination.</p><p>Dinosaurs are extinct today because they lacked opposable thumbs and the brainpower to build a space program.</p>"
  created="2017-10-12T13:27:49.665Z"
  created_at="2017-10-12T13:27:49.665Z"
  metadata={
    Object {
      "author": Object {
        "_id": "5afdbf6c44d74b4e924f1d09",
        "bucket": "5afdbcd115bef94f7421eb0d",
        "content": "<p>Something about John...</p>",
        "created": "2017-10-12T13:27:49.665Z",
        "created_at": "2017-10-12T13:27:49.665Z",
        "metadata": Object {
          "image": Object {
            "imgix_url": "https://cosmic-s3.imgix.net/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
            "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
          },
        },
        "metafields": Array [
          Object {
            "children": null,
            "imgix_url": "https://cosmic-s3.imgix.net/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
            "key": "image",
            "title": "Image",
            "type": "file",
            "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
            "value": "9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
          },
        ],
        "modified_at": "2018-05-17T17:44:12.977Z",
        "published_at": "2018-05-17T17:44:12.977Z",
        "slug": "john-doe",
        "status": "published",
        "title": "John Doe",
        "type_slug": "authors",
      },
      "hero": Object {
        "imgix_url": "https://cosmic-s3.imgix.net/56363630-af74-11e7-b864-313f959a776e-react-blog.jpg",
        "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/56363630-af74-11e7-b864-313f959a776e-react-blog.jpg",
      },
      "teaser": "<p>I don&#39;t know what you could say about a day in which you have seen four beautiful sunsets.</p><p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth. I didn&#39;t feel like a giant. I felt very, very small....</p>",
    }
  }
  metafields={
    Array [
      Object {
        "children": null,
        "imgix_url": "https://cosmic-s3.imgix.net/56363630-af74-11e7-b864-313f959a776e-react-blog.jpg",
        "key": "hero",
        "title": "Hero",
        "type": "file",
        "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/56363630-af74-11e7-b864-313f959a776e-react-blog.jpg",
        "value": "56363630-af74-11e7-b864-313f959a776e-react-blog.jpg",
      },
      Object {
        "children": null,
        "key": "teaser",
        "title": "Teaser",
        "type": "html-textarea",
        "value": "<p>I don&#39;t know what you could say about a day in which you have seen four beautiful sunsets.</p><p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth. I didn&#39;t feel like a giant. I felt very, very small....</p>",
      },
      Object {
        "children": null,
        "key": "author",
        "object": Object {
          "_id": "5afdbf6c44d74b4e924f1d09",
          "bucket": "5afdbcd115bef94f7421eb0d",
          "content": "<p>Something about John...</p>",
          "created": "2017-10-12T13:27:49.665Z",
          "created_at": "2017-10-12T13:27:49.665Z",
          "metadata": Object {
            "image": Object {
              "imgix_url": "https://cosmic-s3.imgix.net/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
              "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
            },
          },
          "metafields": Array [
            Object {
              "children": null,
              "imgix_url": "https://cosmic-s3.imgix.net/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
              "key": "image",
              "title": "Image",
              "type": "file",
              "url": "https://s3-us-west-2.amazonaws.com/cosmicjs/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
              "value": "9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg",
            },
          ],
          "modified_at": "2018-05-17T17:44:12.977Z",
          "published_at": "2018-05-17T17:44:12.977Z",
          "slug": "john-doe",
          "status": "published",
          "title": "John Doe",
          "type_slug": "authors",
        },
        "object_type": "authors",
        "title": "Author",
        "type": "object",
        "value": "5afdbf6c44d74b4e924f1d09",
      },
    ]
  }
  modified_at="2017-10-12T17:39:54.476Z"
  published_at="2018-05-17T17:44:12.979Z"
  slug="a-wonderful-blog-post-about-earth"
  status="published"
  title="A Wonderful Blog Post About Earth"
  type_slug="posts"
>
  <li>
    <button
      className={Object {}}
      onClick={[Function]}
    >
      <div>
        <WithStyles(Typography)
          variant="headline"
        >
          <Typography
            align="inherit"
            classes={
              Object {
                "alignCenter": "MuiTypography-alignCenter-14",
                "alignJustify": "MuiTypography-alignJustify-16",
                "alignLeft": "MuiTypography-alignLeft-13",
                "alignRight": "MuiTypography-alignRight-15",
                "body1": "MuiTypography-body1-10",
                "body2": "MuiTypography-body2-9",
                "button": "MuiTypography-button-12",
                "caption": "MuiTypography-caption-11",
                "colorError": "MuiTypography-colorError-24",
                "colorInherit": "MuiTypography-colorInherit-20",
                "colorPrimary": "MuiTypography-colorPrimary-21",
                "colorSecondary": "MuiTypography-colorSecondary-22",
                "colorTextSecondary": "MuiTypography-colorTextSecondary-23",
                "display1": "MuiTypography-display1-5",
                "display2": "MuiTypography-display2-4",
                "display3": "MuiTypography-display3-3",
                "display4": "MuiTypography-display4-2",
                "gutterBottom": "MuiTypography-gutterBottom-18",
                "headline": "MuiTypography-headline-6",
                "noWrap": "MuiTypography-noWrap-17",
                "paragraph": "MuiTypography-paragraph-19",
                "root": "MuiTypography-root-1",
                "subheading": "MuiTypography-subheading-8",
                "title": "MuiTypography-title-7",
              }
            }
            color="default"
            gutterBottom={false}
            headlineMapping={
              Object {
                "body1": "p",
                "body2": "aside",
                "display1": "h1",
                "display2": "h1",
                "display3": "h1",
                "display4": "h1",
                "headline": "h1",
                "subheading": "h3",
                "title": "h2",
              }
            }
            noWrap={false}
            paragraph={false}
            variant="headline"
          >
            <h1
              className="MuiTypography-root-1 MuiTypography-headline-6"
            >
              A Wonderful Blog Post About Earth
            </h1>
          </Typography>
        </WithStyles(Typography)>
        <WithStyles(Typography)
          component="span"
          variant="body2"
        >
          <Typography
            align="inherit"
            classes={
              Object {
                "alignCenter": "MuiTypography-alignCenter-14",
                "alignJustify": "MuiTypography-alignJustify-16",
                "alignLeft": "MuiTypography-alignLeft-13",
                "alignRight": "MuiTypography-alignRight-15",
                "body1": "MuiTypography-body1-10",
                "body2": "MuiTypography-body2-9",
                "button": "MuiTypography-button-12",
                "caption": "MuiTypography-caption-11",
                "colorError": "MuiTypography-colorError-24",
                "colorInherit": "MuiTypography-colorInherit-20",
                "colorPrimary": "MuiTypography-colorPrimary-21",
                "colorSecondary": "MuiTypography-colorSecondary-22",
                "colorTextSecondary": "MuiTypography-colorTextSecondary-23",
                "display1": "MuiTypography-display1-5",
                "display2": "MuiTypography-display2-4",
                "display3": "MuiTypography-display3-3",
                "display4": "MuiTypography-display4-2",
                "gutterBottom": "MuiTypography-gutterBottom-18",
                "headline": "MuiTypography-headline-6",
                "noWrap": "MuiTypography-noWrap-17",
                "paragraph": "MuiTypography-paragraph-19",
                "root": "MuiTypography-root-1",
                "subheading": "MuiTypography-subheading-8",
                "title": "MuiTypography-title-7",
              }
            }
            color="default"
            component="span"
            gutterBottom={false}
            headlineMapping={
              Object {
                "body1": "p",
                "body2": "aside",
                "display1": "h1",
                "display2": "h1",
                "display3": "h1",
                "display4": "h1",
                "headline": "h1",
                "subheading": "h3",
                "title": "h2",
              }
            }
            noWrap={false}
            paragraph={false}
            variant="body2"
          >
            <span
              className="MuiTypography-root-1 MuiTypography-body2-9"
            >
              <p
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "<p>I don&#39;t know what you could say about a day in which you have seen four beautiful sunsets.</p><p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth. I didn&#39;t feel like a giant. I felt very, very small....</p>",
                  }
                }
              />
            </span>
          </Typography>
        </WithStyles(Typography)>
        <WithStyles(Typography)
          variant="subheading"
        >
          <Typography
            align="inherit"
            classes={
              Object {
                "alignCenter": "MuiTypography-alignCenter-14",
                "alignJustify": "MuiTypography-alignJustify-16",
                "alignLeft": "MuiTypography-alignLeft-13",
                "alignRight": "MuiTypography-alignRight-15",
                "body1": "MuiTypography-body1-10",
                "body2": "MuiTypography-body2-9",
                "button": "MuiTypography-button-12",
                "caption": "MuiTypography-caption-11",
                "colorError": "MuiTypography-colorError-24",
                "colorInherit": "MuiTypography-colorInherit-20",
                "colorPrimary": "MuiTypography-colorPrimary-21",
                "colorSecondary": "MuiTypography-colorSecondary-22",
                "colorTextSecondary": "MuiTypography-colorTextSecondary-23",
                "display1": "MuiTypography-display1-5",
                "display2": "MuiTypography-display2-4",
                "display3": "MuiTypography-display3-3",
                "display4": "MuiTypography-display4-2",
                "gutterBottom": "MuiTypography-gutterBottom-18",
                "headline": "MuiTypography-headline-6",
                "noWrap": "MuiTypography-noWrap-17",
                "paragraph": "MuiTypography-paragraph-19",
                "root": "MuiTypography-root-1",
                "subheading": "MuiTypography-subheading-8",
                "title": "MuiTypography-title-7",
              }
            }
            color="default"
            gutterBottom={false}
            headlineMapping={
              Object {
                "body1": "p",
                "body2": "aside",
                "display1": "h1",
                "display2": "h1",
                "display3": "h1",
                "display4": "h1",
                "headline": "h1",
                "subheading": "h3",
                "title": "h2",
              }
            }
            noWrap={false}
            paragraph={false}
            variant="subheading"
          >
            <h3
              className="MuiTypography-root-1 MuiTypography-subheading-8"
            >
              <img
                alt=""
                height="16px"
                src="https://cosmic-s3.imgix.net/9a0a85b0-23f5-11e7-875c-3f5dc9c15c2b-male.jpg?w=16&h=16&fit=crop"
                width="16px"
              />
              John Doe
               ‧ May 17, 2018
            </h3>
          </Typography>
        </WithStyles(Typography)>
      </div>
      <Hidden
        implementation="js"
        lgDown={false}
        lgUp={false}
        mdDown={false}
        mdUp={false}
        only="xs"
        smDown={false}
        smUp={false}
        xlDown={false}
        xlUp={false}
        xsDown={false}
        xsUp={false}
      >
        <WithTheme(WithWidth(HiddenJs))
          lgDown={false}
          lgUp={false}
          mdDown={false}
          mdUp={false}
          only="xs"
          smDown={false}
          smUp={false}
          xlDown={false}
          xlUp={false}
          xsDown={false}
          xsUp={false}
        >
          <WithWidth(HiddenJs)
            lgDown={false}
            lgUp={false}
            mdDown={false}
            mdUp={false}
            only="xs"
            smDown={false}
            smUp={false}
            theme={
              Object {
                "breakpoints": Object {
                  "between": [Function],
                  "down": [Function],
                  "keys": Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ],
                  "only": [Function],
                  "up": [Function],
                  "values": Object {
                    "lg": 1280,
                    "md": 960,
                    "sm": 600,
                    "xl": 1920,
                    "xs": 0,
                  },
                  "width": [Function],
                },
                "direction": "ltr",
                "mixins": Object {
                  "gutters": [Function],
                  "toolbar": Object {
                    "@media (min-width:0px) and (orientation: landscape)": Object {
                      "minHeight": 48,
                    },
                    "@media (min-width:600px)": Object {
                      "minHeight": 64,
                    },
                    "minHeight": 56,
                  },
                },
                "overrides": Object {},
                "palette": Object {
                  "action": Object {
                    "active": "rgba(0, 0, 0, 0.54)",
                    "disabled": "rgba(0, 0, 0, 0.26)",
                    "disabledBackground": "rgba(0, 0, 0, 0.12)",
                    "hover": "rgba(0, 0, 0, 0.08)",
                    "hoverOpacity": 0.08,
                    "selected": "rgba(0, 0, 0, 0.14)",
                  },
                  "augmentColor": [Function],
                  "background": Object {
                    "default": "#fafafa",
                    "paper": "#fff",
                  },
                  "common": Object {
                    "black": "#000",
                    "white": "#fff",
                  },
                  "contrastThreshold": 3,
                  "divider": "rgba(0, 0, 0, 0.12)",
                  "error": Object {
                    "contrastText": "#fff",
                    "dark": "#d32f2f",
                    "light": "#e57373",
                    "main": "#f44336",
                  },
                  "getContrastText": [Function],
                  "grey": Object {
                    "100": "#f5f5f5",
                    "200": "#eeeeee",
                    "300": "#e0e0e0",
                    "400": "#bdbdbd",
                    "50": "#fafafa",
                    "500": "#9e9e9e",
                    "600": "#757575",
                    "700": "#616161",
                    "800": "#424242",
                    "900": "#212121",
                    "A100": "#d5d5d5",
                    "A200": "#aaaaaa",
                    "A400": "#303030",
                    "A700": "#616161",
                  },
                  "primary": Object {
                    "contrastText": "#fff",
                    "dark": "#303f9f",
                    "light": "#7986cb",
                    "main": "#3f51b5",
                  },
                  "secondary": Object {
                    "contrastText": "#fff",
                    "dark": "#c51162",
                    "light": "#ff4081",
                    "main": "#f50057",
                  },
                  "text": Object {
                    "disabled": "rgba(0, 0, 0, 0.38)",
                    "hint": "rgba(0, 0, 0, 0.38)",
                    "primary": "rgba(0, 0, 0, 0.87)",
                    "secondary": "rgba(0, 0, 0, 0.54)",
                  },
                  "tonalOffset": 0.2,
                  "type": "light",
                },
                "props": Object {},
                "shadows": Array [
                  "none",
                  "0px 1px 3px 0px rgba(0, 0, 0, 0.2),0px 1px 1px 0px rgba(0, 0, 0, 0.14),0px 2px 1px -1px rgba(0, 0, 0, 0.12)",
                  "0px 1px 5px 0px rgba(0, 0, 0, 0.2),0px 2px 2px 0px rgba(0, 0, 0, 0.14),0px 3px 1px -2px rgba(0, 0, 0, 0.12)",
                  "0px 1px 8px 0px rgba(0, 0, 0, 0.2),0px 3px 4px 0px rgba(0, 0, 0, 0.14),0px 3px 3px -2px rgba(0, 0, 0, 0.12)",
                  "0px 2px 4px -1px rgba(0, 0, 0, 0.2),0px 4px 5px 0px rgba(0, 0, 0, 0.14),0px 1px 10px 0px rgba(0, 0, 0, 0.12)",
                  "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 5px 8px 0px rgba(0, 0, 0, 0.14),0px 1px 14px 0px rgba(0, 0, 0, 0.12)",
                  "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.14),0px 1px 18px 0px rgba(0, 0, 0, 0.12)",
                  "0px 4px 5px -2px rgba(0, 0, 0, 0.2),0px 7px 10px 1px rgba(0, 0, 0, 0.14),0px 2px 16px 1px rgba(0, 0, 0, 0.12)",
                  "0px 5px 5px -3px rgba(0, 0, 0, 0.2),0px 8px 10px 1px rgba(0, 0, 0, 0.14),0px 3px 14px 2px rgba(0, 0, 0, 0.12)",
                  "0px 5px 6px -3px rgba(0, 0, 0, 0.2),0px 9px 12px 1px rgba(0, 0, 0, 0.14),0px 3px 16px 2px rgba(0, 0, 0, 0.12)",
                  "0px 6px 6px -3px rgba(0, 0, 0, 0.2),0px 10px 14px 1px rgba(0, 0, 0, 0.14),0px 4px 18px 3px rgba(0, 0, 0, 0.12)",
                  "0px 6px 7px -4px rgba(0, 0, 0, 0.2),0px 11px 15px 1px rgba(0, 0, 0, 0.14),0px 4px 20px 3px rgba(0, 0, 0, 0.12)",
                  "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 12px 17px 2px rgba(0, 0, 0, 0.14),0px 5px 22px 4px rgba(0, 0, 0, 0.12)",
                  "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 13px 19px 2px rgba(0, 0, 0, 0.14),0px 5px 24px 4px rgba(0, 0, 0, 0.12)",
                  "0px 7px 9px -4px rgba(0, 0, 0, 0.2),0px 14px 21px 2px rgba(0, 0, 0, 0.14),0px 5px 26px 4px rgba(0, 0, 0, 0.12)",
                  "0px 8px 9px -5px rgba(0, 0, 0, 0.2),0px 15px 22px 2px rgba(0, 0, 0, 0.14),0px 6px 28px 5px rgba(0, 0, 0, 0.12)",
                  "0px 8px 10px -5px rgba(0, 0, 0, 0.2),0px 16px 24px 2px rgba(0, 0, 0, 0.14),0px 6px 30px 5px rgba(0, 0, 0, 0.12)",
                  "0px 8px 11px -5px rgba(0, 0, 0, 0.2),0px 17px 26px 2px rgba(0, 0, 0, 0.14),0px 6px 32px 5px rgba(0, 0, 0, 0.12)",
                  "0px 9px 11px -5px rgba(0, 0, 0, 0.2),0px 18px 28px 2px rgba(0, 0, 0, 0.14),0px 7px 34px 6px rgba(0, 0, 0, 0.12)",
                  "0px 9px 12px -6px rgba(0, 0, 0, 0.2),0px 19px 29px 2px rgba(0, 0, 0, 0.14),0px 7px 36px 6px rgba(0, 0, 0, 0.12)",
                  "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 20px 31px 3px rgba(0, 0, 0, 0.14),0px 8px 38px 7px rgba(0, 0, 0, 0.12)",
                  "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 21px 33px 3px rgba(0, 0, 0, 0.14),0px 8px 40px 7px rgba(0, 0, 0, 0.12)",
                  "0px 10px 14px -6px rgba(0, 0, 0, 0.2),0px 22px 35px 3px rgba(0, 0, 0, 0.14),0px 8px 42px 7px rgba(0, 0, 0, 0.12)",
                  "0px 11px 14px -7px rgba(0, 0, 0, 0.2),0px 23px 36px 3px rgba(0, 0, 0, 0.14),0px 9px 44px 8px rgba(0, 0, 0, 0.12)",
                  "0px 11px 15px -7px rgba(0, 0, 0, 0.2),0px 24px 38px 3px rgba(0, 0, 0, 0.14),0px 9px 46px 8px rgba(0, 0, 0, 0.12)",
                ],
                "spacing": Object {
                  "unit": 8,
                },
                "transitions": Object {
                  "create": [Function],
                  "duration": Object {
                    "complex": 375,
                    "enteringScreen": 225,
                    "leavingScreen": 195,
                    "short": 250,
                    "shorter": 200,
                    "shortest": 150,
                    "standard": 300,
                  },
                  "easing": Object {
                    "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
                    "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
                    "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
                    "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
                  },
                  "getAutoHeightDuration": [Function],
                },
                "typography": Object {
                  "body1": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "0.875rem",
                    "fontWeight": 400,
                    "lineHeight": "1.46429em",
                  },
                  "body2": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "0.875rem",
                    "fontWeight": 500,
                    "lineHeight": "1.71429em",
                  },
                  "button": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "0.875rem",
                    "fontWeight": 500,
                    "textTransform": "uppercase",
                  },
                  "caption": Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "0.75rem",
                    "fontWeight": 400,
                    "lineHeight": "1.375em",
                  },
                  "display1": Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "2.125rem",
                    "fontWeight": 400,
                    "lineHeight": "1.20588em",
                  },
                  "display2": Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "2.8125rem",
                    "fontWeight": 400,
                    "lineHeight": "1.06667em",
                    "marginLeft": "-.02em",
                  },
                  "display3": Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "3.5rem",
                    "fontWeight": 400,
                    "letterSpacing": "-.02em",
                    "lineHeight": "1.30357em",
                    "marginLeft": "-.02em",
                  },
                  "display4": Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "7rem",
                    "fontWeight": 300,
                    "letterSpacing": "-.04em",
                    "lineHeight": "1.14286em",
                    "marginLeft": "-.04em",
                  },
                  "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                  "fontSize": 14,
                  "fontWeightLight": 300,
                  "fontWeightMedium": 500,
                  "fontWeightRegular": 400,
                  "headline": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "1.5rem",
                    "fontWeight": 400,
                    "lineHeight": "1.35417em",
                  },
                  "pxToRem": [Function],
                  "round": [Function],
                  "subheading": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "1rem",
                    "fontWeight": 400,
                    "lineHeight": "1.5em",
                  },
                  "title": Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
                    "fontSize": "1.3125rem",
                    "fontWeight": 500,
                    "lineHeight": "1.16667em",
                  },
                },
                "zIndex": Object {
                  "appBar": 1100,
                  "drawer": 1200,
                  "mobileStepper": 1000,
                  "modal": 1300,
                  "snackbar": 1400,
                  "tooltip": 1500,
                },
              }
            }
            xlDown={false}
            xlUp={false}
            xsDown={false}
            xsUp={false}
          >
            <EventListener
              onResize={[Function]}
              target="window"
            >
              <HiddenJs
                lgDown={false}
                lgUp={false}
                mdDown={false}
                mdUp={false}
                only="xs"
                smDown={false}
                smUp={false}
                width="md"
                xlDown={false}
                xlUp={false}
                xsDown={false}
                xsUp={false}
              >
                <figure>
                  <img
                    alt=""
                    height="140px"
                    src="https://cosmic-s3.imgix.net/56363630-af74-11e7-b864-313f959a776e-react-blog.jpg?w=140&h=140&fit=crop"
                    width="140px"
                  />
                </figure>
              </HiddenJs>
            </EventListener>
          </WithWidth(HiddenJs)>
        </WithTheme(WithWidth(HiddenJs))>
      </Hidden>
    </button>
  </li>
</PostPreview>
`;
